### ISSUE TITLE
Suggestion: Make player hold fishing rod in both hands

### ISSUE THREAD

**Issue by LordEnder-Kitty** [tr7zw/NotEnoughAnimations#179]
just an idea i had. it would make more sense to hold it in both hands because when have you ever seen someone fish with one hand in real life?

**Comment by Sir-Crazynut** [comment]
true, also the actual fishing line doesn't follow the fishing rod with the idle animation, it just hovers in one place. That would also be a nice fix
